modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -8,7 +8,7 @@ Review the following limitations on support for local registries:
  
 
 [IMPORTANT]
-When installing Red Hat OpenShift operators, you cannot use local registries.  
+Local registry for Runtime Fabric works for OpenShift RTF as well as for the purpose of storing the Mule Docker images to install and manage Runtime Fabric for running Mule apps in K8s. Though, when you install the OpenShift operator per se, you cannot use local registries in the installation. 
```

Comment:
The first sentence is a bit long and hard to understand. I didn't find OpenShift RTF in the OpenShift doc, but might have missed it in my search. Is the purpose of this to give different uses for local registry?
Also, is "per se" a specific part of OpenShift operator, or are you just using it for emphasis? We try to avoid latinisms, and in the context there, I don't think it makes sense. I'm not sure I understand this enough to rewrite it, but perhaps you could break the first sentence into two and put it after the limitation (2nd sentence) something like:
You cannot use local registries during the installation of OpenShift operator. However, you can use local registries with OpenShift RTF to store Mule Docker images and install and manage Runtime Fabric Mule apps in Kubernetes.